src/test/java/com/reservation/client/ClientTestFactory.java

```diff
@@ -0,0 +1,21 @@
+package com.reservation.client;
+
+import java.time.Instant;
+import java.util.UUID;
+
+class ClientTestFactory {
+
+    public static final Instant FIXED_INSTANT = Instant.parse("2024-01-01T12:00:00Z");
+    public static final UUID ID = UUID.fromString("07392a26-375d-4794-9eed-44a57c7efd41");
+
+    static Client expectedTestClient() {
```

Comment:
Myślę, czy nie lepiej byłoby to nazwać np:
```suggestion
    static Client validClient() {
```
bo w sumie wiadomo, że to klient testowy, bo używamy go tylko w testach 😁
No i też tak jak już gdzieś wyżej napisałem, ładnie to można użyć z builderami i stworzyć sobie np. nieprawidłowego użytkownika do przetestowania jakiegoś szczególnego przypadku biznesowego.

Reply:
Valid point milordzie :D

Reply:
zastanawialem sie pomiedzy nazwa expectedTestClient a existedTestClient, wybrałem to pierwsze bo ten obiekt to wierna kopia istniejacego rekordu w db testcontainers z pliku resource/db/test/2.1__insert_test_data.sql

INSERT INTO client (id, name, email, created_at)
VALUES ( '07392a26-375d-4794-9eed-44a57c7efd41','client-1', 'client-1@example.com', now());

zeby mozna go bylo nazwac existedTestClient to musiał bym go pobrać z db ale wtedy inne testy były by zależne od operacji pobierania tego rekordu dlatego go tworzę jako nowy obiekt który jest bardziej skonfigurowany niż validClient który mógłby mieć dowolne wartośći